src/chrome/content/rules/OpenMandriva.org.xml

```diff
@@ -0,0 +1,54 @@
+<!--
+	Invalid certificate:
+		archive.openmandriva.org
+		blog2.openmandriva.org
+		downloads.openmandriva.org
+		ftp.openmandriva.org
+		garnet.openmandriva.org
+		ml.openmandriva.org
+		repoclosure.openmandriva.org
+
+	No working URL known:
+		email.openmandriva.org
+		jasper.openmandriva.org
+		listen.openmandriva.org
+		project.openmandriva.org (502)
+		torrents.openmandriva.org
+
+	Secure connection failed:
+		hcl.openmandriva.org
+
+	Different content http/https:
+		www-static.openmandriva.org
+
+-->
+<ruleset name="OpenMandriva.org">
+
+	<target host="openmandriva.org" />
+	<target host="www.openmandriva.org" />
+	<target host="abf.openmandriva.org" />
+	<target host="ask.openmandriva.org" />
+	<target host="auth.openmandriva.org" />
+	<target host="blog.openmandriva.org" />
+	<target host="chwido.openmandriva.org" />
+	<target host="discourse.openmandriva.org" />
+	<target host="discourse-static.openmandriva.org" />
+	<target host="doc.openmandriva.org" />
+	<target host="forum.openmandriva.org" />
+	<target host="forums.openmandriva.org" />
+	<target host="archives.forums.openmandriva.org" />
+	<target host="gallery.openmandriva.org" />
+	<target host="issues.openmandriva.org" />
+	<target host="media.openmandriva.org" />
+	<target host="pad.openmandriva.org" />
+	<target host="sitedev.openmandriva.org" />
+	<target host="static.openmandriva.org" />
+	<target host="stats.openmandriva.org" />
+	<target host="wiki.openmandriva.org" />
```

Comment:
Aren't they already in alphabetical order? Could you point out to specific lines please? I can't find the issue.